docs/testbed/README.testbed.QosRpc.md

```diff
@@ -0,0 +1,65 @@
+# Background
+
+In order to run qos sai test, it's essential to build syncd RPC container, upload relevant artifacts to docker registry, and config test code to access docker registry.
+
+
+# Build your own syncd RPC container image
+
+## understand the build system.
+
+Familiarize yourself with the SONiC build system by reviewing [doc](https://github.com/sonic-net/sonic-buildimage/blob/master/README.md)
+
+## building the syncd RPC Container Image
+
+Follow the build steps in above documentation. Before proceeding to the "build sonic image" step, build the syncd RPC container image for your platforms using below command:
+
+```
+# Build syncd rpc conitainer iamge
+make ENABLE_SYNCD_RPC=y target/docker-syncd-${platform_rpc}-rpc.gz
+```
+
+Replace ${platform_rpc} with the appropriate value for your platform:
+
+- Barefoot: bfn
+- Broadcom: brcm
+- Centec: centec
+- Mellanox: mlnx
+- Nephos: nephos
+
+# Setup your docker registry
+
+For detailed instructions on establishing and managing your Docker registry, refer to [Docker Registry Documentation](https://docs.docker.com/registry/)
+
+# Access docker registry in QoS SAI test script
+
+Modify [docker_registry.yml](ansible/vars/docker_registry.yml) to fill docker registry link, username and password. e.g:
```

Comment:
(ansible/vars/docker_registry.yml) should be a link in markdown, but seems it is invalid here.

Reply:
updated